projects/plugins/crm/includes/ZeroBSCRM.Core.Localisation.php

```diff
@@ -314,17 +313,18 @@ function zeroBSCRM_date_forceEN( $time = -1 ) {
 		// (Month names are localised, causing a mismatch here (Italian etc.))
 		// ... so we translate:
 		// d F Y H:i:s (date - not locale based)
-		// https://www.php.net/manual/en/function.date.php
 		// ... into
-		// dd MMMM yyyy HH:mm:ss (IntlDateFormatter - locale based date)
-		// (https://www.php.net/manual/en/class.intldateformatter.php)
+		// d F Y H:i:s (date - locale based date)
+		// (https://www.php.net/manual/en/function.date.php)
```

Comment:
I think this long comment could use some rewriting. It comes across now as "instead of using `date`, we use `date` because `date` isn't affected by the locale issues like `date` is".